src/main/java/net/devtech/arrp/json/loot/JEntry.java

```diff
@@ -1,26 +1,20 @@
 package net.devtech.arrp.json.loot;
 
+import net.devtech.arrp.impl.RuntimeResourcePackImpl;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public class JEntry implements Cloneable {
 	private String type;
 	private String name;
-	private List<String> children;
+	private List<JEntry> children;
 	private Boolean expand;
 	private List<JFunction> functions;
-	private List<JCondition> condition;
+	private List<JCondition> conditions;
 	private Integer weight;
 	private Integer quality;
 
-	public JEntry condition(JCondition condition) {
```

Comment:
ARRP will never break backward compatibility unless absolutely nessesary, deprecate the method instead of removing it

Reply:
`JEntry#condition` still exists, I just moved it down in the source file